### PR TITLE
#158556165 Restructure a ride offer to show important details.

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
**What does this PR do?**
Restructures a ride offer so that it displays important details to passengers as shown on the tasks to be completed below.

**Description of Task to be completed?**
- Rename `user.html` to `rides.html`
- Modify ride offer card on `rides.html` to have : 
    - Area/location of the driver offering the ride
    - Destination showing the point where the journey ends
    - Route the driver is going to take
    - Time of departure for the driver so that passengers can plan their time
    - link to show more details of the ride
- Modify `ridedetails.html` to have:
    - Name of the driver offering the given ride
    - Phone contact for the driver so that passengers can conduct him to plan their journey together.
    - short notes encouraging potential passengers to conduct driver
    - Driver rating from users.
    - Link for passengers to click to join the ride.
- Update CSS to reflect the new layout.

**How should this be manually tested?**
- Clone this repository
- In your terminal, cd ride-my-way
- `git checkout ft-restructure-view-rides-158556165
- Open `rides.html` and `ridedetails.html` using your favorite browser.

**What are the relevant pivotal tracker stories?**
[#158556165](https://www.pivotaltracker.com/story/show/158556165)

**Screenshots**
rides

moredetails

### CODE REVIEW

**Comment by JoshuaOndieki** [comment]
**Area** is ambiguous. I think it will take users quite some time to relate it as referring to the starting point.
My suggestion would be to use **ORIGIN** or **STARTING POINT**

File: ui/rides.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+				    <li class="right"><a href="./profile.html">Profile</a></li>
+				    <li class="icon"><a href="javascript:void(0);" onclick="toggleResponsive()"><i class="fa fa-bars"></i></a></li>
+
+				</ul> 
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col-sm-3"></div>
+			<div class="header"><h1>Scroll down to view more rides.</h1></div>
```
Comment (JoshuaOndieki) [inline ui/rides.html:31]: I think at this point it's a common knowledge that a user needs to scroll down to see more content.
Maybe you could use this limited and valuable space for other information users actually need.

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: ui/rides.html (hunk 1 above)
Comment (meshack-mbuvi) [reply ui/rides.html:31]: point noted.

Comment (meshack-mbuvi) [reply ui/rides.html:31]: What do you think of having *Start point* instead of *starting point*?

**Comment by JoshuaOndieki** [comment]
You've made a lot of changes in one commit.
I wish you could have broken them into smaller changes with more descriptive and precise commit statements.

File: ui/rides.html (hunk 1 above)
Comment (JoshuaOndieki) [reply ui/rides.html:31]: I'm cool with that too :+1:

File: ui/css/main.css [hunk 2]
```diff
@@ -439,19 +360,5 @@ i.fa-plus{
...
-    width: 200px;
-    height: 200px;
-    border-radius: 50%;
-    
-}
-.car_image{
-    width: 100%
-}
 
-/*/* Force table to not be like tables anymore */
```
Comment (JoshuaOndieki) [inline ui/css/main.css]: I believe this commented out block of code is of no use any longer. Consider removing it.

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

**Comment by meshack-mbuvi** [comment]
I have a question, when should you make a commit, are you not supposed to commit when you are done with the change?what do you think is the best practice for committing changes?

File: ui/css/main.css [hunk 3]
```diff
@@ -439,19 +360,5 @@ i.fa-plus{
...
-    
-}
-.car_image{
-    width: 100%
-}
 
-/*/* Force table to not be like tables anymore */ 
-/*@media only screen and (max-width: 500px){*/
-  /*table, thead, tbody, th, td, tr {*/
-      /*display: block; */
```
Comment (JoshuaOndieki) [inline ui/css/main.css]: I believe this commented out block of code is of no use any longer. Consider removing it.

File: ui/css/main.css (hunk 2 above)
Comment (meshack-mbuvi) [reply ui/css/main.css]: Please confirm it is not there on the changes made.

**Comment by JoshuaOndieki** [comment]
I too have similar commits and by reading your commits, I've realized it's quite a huge mistake we are making. Your commit: **Add specific details for a ride** begs the question, **Which specific details*?*
Also you've not just added details to your ride, you've made changes to your CSS.

File: ui/css/main.css (hunk 2 above)
Comment (JoshuaOndieki) [reply ui/css/main.css]: Oh, you removed it in this commit. Sorry about that

**Comment by meshack-mbuvi** [comment]
I see the point. More so when I have made the changes on the CSS. Just curious, does restructure not involve changing the css?

**Comment by JoshuaOndieki** [comment]
Restructure sure does involve CSS

File: ui/rides.html [hunk 4]
```diff
@@ -34,7 +34,7 @@
 		<div class="row">
 			<div class="col-sm-12 col-md-3">
 				<div class="details">
-					<p><span class="label">Area : </span><span> Makongeni, corner two</span></p>
+					<p><span class="label">Start point : </span><span> Makongeni, corner two</span></p>
```
Comment (JoshuaOndieki) [inline ui/rides.html]: :+1:

**Review by JoshuaOndieki — APPROVED** [review]

**Review by LWanjiru — APPROVED** [review]
This looks good.  Kudos!

**Pull request merged**